src/rutujaGDec19/OddEvenSumTest.java

```diff
@@ -0,0 +1,29 @@
+package rutujaGDec19;
+
+public class OddEvenSumTest {
+	static int EvenSum(int x) {
+		int esum = 0;
+		esum = esum + x;
+		return esum;
+	}
+	static int OddSum(int x) {
+		int osum = 0;
+		osum = osum + x;
+		return osum;
+	}
+		public static void main(String[] args) {
+			int Even = 0 , Odd = 0;
```

Comment:
Logic is correct, but try to follow best practice
1) Class Name first letter should be Capital always.
2) variable first letter should be small.
3) method name should be logical, means like if you are finding particular number from array then name should be checkNumberInArray(int number)
      -> method name should be like, analyze capital and small letter combination


All comments not applicable just to share information.